tools/test_utils.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+Transaction_ABI="https://raw.githubusercontent.com/eosnetworkfoundation/mandel-eosjs/main/src/transaction.abi.json"
```

Comment:
Seems it would be better to just copy `transaction.abi.json` into the `test` directory. I don't think we want to setup this dependency.

Reply:
got it , I will use the external dependency that is pulled in with the git-submodule of mandel-eosjs